docs/Using-Fleet/Mobile-device-management.md

```diff
@@ -2,14 +2,58 @@
 
 MDM features are not ready for production and are currently in development. These features are disabled by default.
 
-MDM features allow you to mange macOS updates and macOS settings on your hosts.
+MDM features allow you to manage macOS updates and macOS settings on your hosts.
 
-To use MDM features you have to connect Fleet to Apple Push Certificates Portal:
+## Apple Push Notification Service (APNs)
 
-1. In the Fleet UI, head to the **Settings > Integrations > Mobile device management (MDM)** page. Users with the admin role can access the settings pages.
+To use MDM features you have to connect Fleet to Apple Push Certificates Portal. This can be done via the Fleet UI or the `fleetctl` command-line interface. The result is the same regardless of the method used, you end up with 4 files: an APNs certificate and private key files along with the Simple Certificate Enrollment Protocol (SCEP) certificate and private key files. Make sure you store them securely as you will need them to configure your Fleet instances afterwards (and take note of the email and organization used to generate them, as you will need to use the same ones during renewal). For _renewal_ of an existing but expired (or soon to expire) APNs certificate, see the [APNs Renewal section](#apns-renewal) below.
```

Comment:
```suggestion
To connect Fleet to Apple, get these four files using the Fleet UI or the `fleetctl` command-line interface: An APNs certificate, APNs private, Simple Certificate Enrollment Protocol (SCEP) certificate, and SCEP private key. 

To do this, choose the "Fleet UI" or "fleetctl" method and follow the steps below.
```

I think we can shorten some of this for clarity. Please let me know if I'm cutting anything important.

I moved removed the "store the certs/keys securely" sentence and moved "take note of the email" sentence below.